### DISCUSSION TITLE
1.8.5

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#2222] [New Release]
### Changelog
- Update translation to fix crashes in some languages
- Some UI fixes
- Avoid calling `finishReceiver` for unordered broadcasts
- Clear application profile data before performing dexOpt
- Distinguish update channels when checking updates
- Fix hooking/deoptimizing static methods fails on some Android 13 devices
- Repository shows assets size and download counts
- Fix hooking proxy method
- Init resources hook when calling `hookSystemWideLayout`

### 更新日志
- 更新翻译以修复部分语言下的崩溃
- 一些用户界面的修复
- 避免为无序广播调用 `finishReceiver`
- 执行 dexOpt 之前清空应用 profile 数据
- 检查更新时候区分更新通道
- 修复某些 Android 13 设备上静态方法挂钩/反优化失败
- 仓库显示附件大小和下载次数
- 修复代理方法挂钩
- 调用 `hookSystemWideLayout` 时候初始化资源钩子

### Contributors
@yujincheng08 @vvb2060 @Howard20181 @green1052 @5ec1cff 

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.8.5'>1.8.5</a>.</em>

**Comment by FoxhoundX** [comment]


**Reply by Howard20181** [reply]
No logs, no fixed.
My guess is that it's a problem with the luckypatcher.

**Comment by Tuanzi010** [comment]
Sir, what is the minimum version of Magisk supported by the LSP framework?

**Reply by Howard20181** [reply]
https://github.com/LSPosed/LSPosed/blob/cd6e9473970811262e824d0a4e22e3a22878bc84/magisk-loader/magisk_module/util_functions.sh#L21-L42

**Reply by Tuanzi010** [reply]
ok, thanks man

**Comment by jimger** [comment]
Is it expected that it doesn't work on oxygen OS android 13?
It seems it is running ok but none of the modules can see it...

**Reply by Howard20181** [reply]
We have never received feedback that it doesn't work, if there is a problem you should open a new [issue](https://github.com/LSPosed/LSPosed/issues).

**Reply by jimger** [reply]
Ok will do. Need to is tall debug version as it seems.... BTW half works. Twifuck and gphotos work, but hidemyapp and updatelocker and chatheadenabler don't :-(

**Comment by RajPratapSinghVIP** [comment]
**Dex Optimizer Wrapper
Unsupported (Android Version Unsatisfied)**

_I'm using android 9
Nokia 8 (TA-1004)
Using official room_

**Why am i facing this problem any solution??**
_(I'm alredy tryed formate phone method and TWRP formate also but no profit {pluse i restored my room result nothing})_

**Reply by Howard20181** [reply]
It only supports Android 10+

**Reply by RajPratapSinghVIP** [reply]
Thanks then i need to update
